### PR TITLE
Add not support tag

### PR DESCRIPTION
**Pull request by TheKodeToad**
this tag is my masterpiece

### CODE REVIEW

File: tags/not_support.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: 🚨 This is not a support channel! 🚨 
+color: orange
+---
+
```
Comment (KTrain5169) [inline tags/not_support.md:5]:
```suggestion
This channel is not designated as a support channel!

To find support for your problem(s), go to the relevant channel from below:

```

File: tags/not_support.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<:waaa_prism:1041361839528878160> **Problem in launcher**
+👉 <#1032130809055940690> 👈 
+
+<:fried:1032296608768995368> **Problem with mods**
```
Comment (KTrain5169) [inline tags/not_support.md:9]:
```suggestion
<:fried:1032296608768995368> **Problem with using mods in-game**
```

**Review by KTrain5169 — CHANGES_REQUESTED** [review]

File: tags/not_support.md (hunk 2 above)
Comment (TheKodeToad) [reply tags/not_support.md:9]: That would make it seem like it's not for crashes

**Review by TayouVR — APPROVED** [review]

File: tags/not_support.md (hunk 1 above)
Comment (TayouVR) [reply tags/not_support.md:5]: I agree, this isn't needed with the title.

**Review by getchoo — APPROVED** [review]

**Comment by getchoo** [comment]
> this tag is my masterpiece

I agree it's beautiful

**Pull request merged**